test-benchmark-visualizer/test-benchmark-sparkline.html

```diff
@@ -0,0 +1,253 @@
+`--test-benchmark-sparkline-dark`         | Background color at the bottom of the sparkline. | `--paper-blue-grey-700`
+`--test-benchmark-sparkline-axis-color`   | The axis color for the sparkline.                | `--paper-blue-grey-500`
+`--test-benchmark-sparkline-column-color` | The column color for the sparkline.              | `--paper-blue-grey-100`
+
+@demo demo/index.html
+-->
+
+<dom-module id="test-benchmark-sparkline">
+  <template>
+    <style>
+      :host {
+        @apply(--layout);
+        @apply(--layout-flex);
+        @apply(--layout-vertical);
+        @apply(--paper-font-common-base);
+        @apply(--shadow-elevation-2dp);
+
+        box-sizing: border-box;
+        border-radius: 2px;
+        overflow: hidden;
+
+
+        color: #fff;
+
+        background-color: var(--test-benchmark-sparkline-light, --paper-blue-grey-500);
+      }
+
+      #chart {
+        flex: 1 0 60px;
+        padding: 1em 1em;
+        box-sizing: border-box;
+        background-color: var(--test-benchmark-sparkline-dark, --paper-blue-grey-700);
+      }
+
+      .x-axis-path {
+        stroke-width: 1px;
+        stroke: var(--test-benchmark-sparkline-axis-color, --paper-blue-grey-500);
+      }
+
+      .column {
+        fill: var(--test-benchmark-sparkline-column-color, --paper-blue-grey-100);
+      }
+
+      #stats {
+        padding: 0.5em;
+        text-align: center;
+      }
+
+      #stats h1 {
+        font-size: 18px;
+        font-weight: normal;
+      }
+
+      #stats h2 > span {
+        font-size: 16px;
+        font-weight: normal;
+      }
+
+      #stats * {
+        margin: 8px 0;
+      }
+
+      #stats :first-child {
+        margin-top: 0;
+      }
+
+      #stats :last-child {
+        margin-bottom: 0;
+      }
+
+      svg {
+        width: 100%;
+        height: 30px;
+      }
+    </style>
+
+    <div id="stats">
+      <h1>[[name]]</h1>
+      <h2>[[average]]ms <span>± [[fixedStandardDeviation]]</span></h2>
+    </div>
+    <div id="chart"></div>
+  </template>
+</dom-module>
+<script>
+Polymer({
+  is: 'test-benchmark-sparkline',
+
+  behaviors: [
+    Polymer.IronResizableBehavior
+  ],
+
+  properties: {
+
+    /**
+     * The `PerformanceTest` instance that this sparkline is visualizing.
+     */
+    test: {
+      type: Object,
+      observer: 'update'
+    },
+
+    /**
+     * The name of the test being visualized.
+     */
+    name: {
+      type: String,
+      readOnly: true
+    },
+
+    /**
+     * The avarage of all run results for the test being visualized.
+     */
+    average: {
+      type: Number,
+      readOnly: true
+    },
+
+    /**
+     * The fixed standard deviation for all results of the test being
+     * visualized.
+     */
+    fixedStandardDeviation: {
+      type: Number,
+      readOnly: true
+    }
+  },
+
+  listeners: {
+    'iron-resize': 'update'
+  },
+
+  created: function() {
+    this.x = d3.scale.linear();
```

Comment:
Will change `public` to `protected`. This is an artifact of porting from the initial prototype (which was made almost a year ago).

`created` made more sense to me since it is property initialization, and I have been doing that in `created`. WDYT?